src/Core/System/SystemConfig/Command/ConfigSet.php

```diff
@@ -29,6 +29,8 @@ protected function configure(): void
         $this
             ->addArgument('key', InputArgument::REQUIRED)
             ->addArgument('value', InputArgument::REQUIRED)
+            /* @deprecated tag:v6.5.0 - Will be made REQUIRED in v6.5.0 */
+            ->addArgument('type', InputArgument::OPTIONAL, 'string|bool|boolean', 'string')
```

Comment:
Maybe we could allow also json. It could theoretically also solve the bool conversation problem. What do you think

Reply:
we also should add `int` and `float`

Reply:
will post again, how it was solved in SW5 :grin: https://github.com/shopware/shopware/blob/5.7/engine/Shopware/Commands/ConfigSetCommand.php#L61-L63

if decode is used, it will automatically be the right type